{"uuid":"7ae510ea-17a5-418e-984c-c1b2e9f0e7ae","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["161b4eed-c58c-40c5-8eaf-ecbba5983049"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"c068f6d1-b123-41ac-b602-372cdd728389-attachment","type":"image/png"},{"name":"Myscreenshot","source":"aaf04b91-0f25-4f12-bacd-c14c6d505ff9-attachment"}],"parameters":[],"start":1696887714770,"stop":1696887714963},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887714963,"stop":1696887714963},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887714964,"stop":1696887714964},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887714964,"stop":1696887714964}],"start":1696887714762,"stop":1696887714965}